include/taskolib/SequenceManager.h

```diff
@@ -219,6 +224,16 @@ class SequenceManager
      */
     static UniqueId create_unique_id(const std::vector<SequenceOnDisk>& sequences);
 
+    /**
+     * Stage all files git can find in a repository.
+     * A file-glob can be specified to find only the files that are wanted.
```

Comment:
The way we configured Doxygen, everything up to the first period (".") is treated like a `\brief` comment. The empty line does not matter one way or another for the generated HTML files, but it can sometimes make the comment more readable in the code.